setup.py

```diff
@@ -68,9 +72,10 @@
         # requirements files see:
         # https://packaging.python.org/en/latest/requirements.html
         install_requires=[
-            'Django>=1.6', 
-            'ims_lti_py', 
-            'django-braces'
+            'Django>=2.0,<3.0', 
```

Comment:
Based on the [backwards incompatible changes in Django 3.0](https://docs.djangoproject.com/en/3.1/releases/3.0/#backwards-incompatible-changes-in-3-0), I don't think any changes will be required to support Django 3.0 unless I'm missing something. Since the scope of this PR is to upgrade to Django 2.2, I'm not advocating a change, just speculating that we might be able to relax the version constraint (e.g. `Django>=2.0`).